lambdas/subscribe/main.go

```diff
@@ -25,14 +29,35 @@ type Request struct {
 	PhoneNumber string `json:"phoneNumber"`
 }
 
+var mongoDBURI = os.Getenv("MONGODB_URI")
+
+func init() {
+	mongoDBURI = decrypt(mongoDBURI)
+}
+
+func decrypt(encrypted string) string {
+	kmsClient := kms.New(session.New())
+	decodedBytes, err := base64.StdEncoding.DecodeString(encrypted)
+	if err != nil {
+		panic(err)
+	}
+	input := &kms.DecryptInput{
+		CiphertextBlob: decodedBytes,
+	}
+	response, err := kmsClient.Decrypt(input)
+	if err != nil {
+		panic(err)
+	}
+	// Plaintext is a byte array, so convert to string
+	return string(response.Plaintext[:])
+}
+
 // Handler function for lambda
 func Handler(ctx context.Context, request events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
 	//				START MONGODB SETUP					//
-	client, err := mongo.Connect(ctx, options.Client().ApplyURI(
-		os.Getenv("MONGODB_URI"),
-	))
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI(mongoDBURI))
 	if err != nil {
-		log.Fatal("Connection error:", err)
+		log.Println("Connection error:", err)
```

Comment:
`log.Fatal` terminates the program which means the `return` never happens. Made it `log.Println` to fix this.